### ISSUE TITLE
[Minor] Pokemon database is out of date

### ISSUE THREAD

**Issue by grizeldi** [cookkkie/mee6#80]
The search function for pokemon does not have latest generation (Sun/Moon) available.
!pokemon solgaleo returns "I didn't find anything"

**Comment by MrRedAmber** [comment]
`https://veekun.com/dex/pokemon/search`  This is the URL that the Mee6 !pokemon command interacts with when searching for a Pokemon. Mee6 doesn't upkeep it's own Pokemon database, it's all handled through Veekun.com using BeautifulSoup to modify HTML form elements on the page to create an automated request sequence. The request is then posted to the server and any data found relating to that Pokemon name will be returned to you in Mee6.

TL;DR - Veekun.com is responsible if any pokemon could not be found, not Mee6. If any other pokemon can be found from Sun/Moon on Mee6 then let me know and I will look deeper into this issue.
